WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/RefundConfirmationViewModel.swift

```diff
@@ -191,15 +191,14 @@ private extension RefundConfirmationViewModel {
             case .some(.cardPresent(let cardDetails)), .some(.interacPresent(let cardDetails)):
                 return PaymentDetailsRow(cardIcon: cardDetails.brand.icon,
                                          cardIconAspectHorizontal: cardDetails.brand.iconAspectHorizontal,
-                                         paymentGateway: details.order.paymentMethodTitle,
+                                         paymentGateway: paymentGatewayMethodTitle(),
                                          paymentMethodDescription: cardDetails.brand.cardDescription(last4: cardDetails.last4),
                                          accessibilityDescription: cardDetails.brand.cardAccessibilityDescription(last4: cardDetails.last4))
             default:
                 return SimpleTextRow(text: details.order.paymentMethodTitle)
             }
         } else {
-            return TitleAndBodyRow(title: Localization.manualRefund(via: details.order.paymentMethodTitle),
-                                   body: Localization.refundWillNotBeIssued(paymentMethod: details.order.paymentMethodTitle))
+            return TitleAndBodyRow(title: Localization.manualRefund, body: Localization.refundWillNotBeIssued)
```

Comment:
❓ Do we need to be more specific when a payment method is available? E.g:
- if the payment method is Stripe: we can display "Manual Refund via Stripe"
- otherwise: display "Manual Refund"

It seems that on Android they're doing [something similar here](https://github.com/woocommerce/woocommerce-android/blob/cf60636d859ff421269b1afce36f4fa5f21de657/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/refunds/RefundDetailViewModel.kt#L142-L153).

Reply:
That's something that crossed my mind as it is what we're doing in iOS, however, I was unable to replicate it in Android: Every time I tried to go through the refund process for this specific case ( a payment gateway is associated with the Order, but an automatic refund is not possible ) I've always got the generic message back. I'm going to check with other gateways and see the results.